### ISSUE TITLE
Fix broken travis

### ISSUE THREAD

**Issue by mattias-p** [dotse/zonemaster-gui#203]
Zonemaster::Backend 2.0.0 broke the Zonemaster::GUI travis script. The travis script needs to be fixed to use the new names.

**Comment by matsduf** [comment]
```
$ export BASE_DIR=`dirname "$TRAVIS_BUILD_DIR"`
$ export ZONEMASTER_BACKEND_CONFIG_FILE=$BASE_DIR/zonemaster-backend/share/travis_postgresql_backend_config.ini
$ export URL=localhost:5080
$ starman --error-log=/tmp/zm_front.log --workers=2 --listen=127.0.0.1:5080 -I./lib zm_app/bin/app.pl &
$ cd ..
$ git clone https://github.com/dotse/zonemaster-backend.git
$ cd zonemaster-backend
$ cpanm --quiet --installdeps --notest .
$ if [[ "$TARGET" == "PostreSQL" ]]; then psql -c "create user travis_zonemaster WITH PASSWORD 'travis_zonemaster';" -U postgres; fi
$ if [[ "$TARGET" == "PostreSQL" ]]; then psql -c 'create database travis_zonemaster OWNER travis_zonemaster;' -U postgres; fi
$ if [[ "$TARGET" == "PostreSQL" ]]; then cpanm --quiet --notest DBD::Pg; fi
$ if [[ "$TARGET" == "PostreSQL" ]]; then perl -I./lib ./script/create_db_postgresql_9.3.pl; fi
$ starman --error-log=/tmp/zm_back.log --workers=2 --listen=127.0.0.1:5000 -I./lib/ ./script/zonemaster_webbackend.psgi &
$ perl -I./lib ./script/zm_wb_daemon start

Can't open perl script "./script/zm_wb_daemon": No such file or directory

```

**Comment by matsduf** [comment]
Resolved by PR #205
